lib/widget/input/form_textfield.dart

```diff
@@ -218,6 +222,8 @@ class TextInputState extends FormFieldWidgetState<BaseTextInput>
 
   @override
   void initState() {
+    widget.textController =
+        TextEditingController(text: widget.controller.value);
```

Comment:
Why initialize a new controller here instead of setting the value? Based on comment from line 92, textController should be source of truth for value?

Reply:
It initializes the text editing controller with the setter value (controller.value). 

It's because previously we had `textController.text` for setters and getters, so the value is added to the TextField directly. 

Now I'm using `controller.value `for setters and getters, so I need to reinitialize the text editing controller with this `controller.value` in the init method (226 line)

See line number 33 and 103 in the files changed tab

Reply:
Hmmm @vusters what do you think about this? It would be inverting the flow of data.